torchao/quantization/subclass.py

```diff
@@ -134,14 +139,21 @@ def __torch_function__(cls, func, types, args=(), kwargs=None):
 
     @classmethod
     def __torch_dispatch__(cls, func, types, args, kwargs):
+        # Note: we only added cpu path here for 8da4w, this is for executorch, in the future
+        # 1. we'll add cpu/cuda version (int4mm etc.)
+        # 2. we'll need to hide the 8da4w executorch version under things like layouts (we also have multiple impl for cpu kernel as Michael mentioned), so it will be something like
+        #   cpu device + et laytout --> gives current 8da4w executorch representation
```

Comment:
just for me to learn: wdym by layout here?

Reply:
meaning this: https://github.com/pytorch/pytorch/blob/main/c10/core/Layout.h#L10 just another dimension on top of device

Reply:
Thank you!